src/styles/use_case.css

```diff
@@ -0,0 +1,62 @@
+.use-case-page {
+  .title {
+    font-size: 32px;
+    font-weight: bold;
+  }
+
+  p {
+    margin-bottom: 0;
+  }
+
+  .col-12 {
+    padding: 20px !important;
+  }
+}
+
+@media (max-width: 768px) {
+  .use-case-page {
+    .title,
+    p {
+      text-align: center !important;
+    }
+
+    .col-12 {
+      padding: 16px !important;
+    }
+  }
+}
+
+.use-case-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+
+  .icon {
+    width: 100%;
+    display: flex;
+    justify-content: start;
+    margin-bottom: 4px;
+
+    img {
+      width: 54px;
+      height: 54px;
+    }
+  }
+
+  h2 {
+    font-size: 20px;
+    font-weight: bold;
+    margin-bottom: 4px;
+  }
+}
+
+@media (max-width: 768px) {
+  .use-case-card:not(.align-left) {
```

Comment:
Why not center-align for `Still need a reason?`? Below looks inconsistent

<img width="462" alt="image" src="https://github.com/user-attachments/assets/82af4358-47d1-42c9-bedb-a27bb5a96872">

Reply:
`Still nee a reason` part is designed to be left aligned
![image](https://github.com/user-attachments/assets/e9c4ffd1-698a-466a-95a4-62e9bcf3b762)